.github/workflows/release.yml

```diff
@@ -13,10 +13,10 @@ jobs:
       GIT_USER_EMAIL: behlp@unityfoundation.io
```

Comment:
Change to `GIT_USER_NAME: grails-build` and `GIT_USER_EMAIL: grails-build@users.noreply.github.com`